src/microbe_stage/PatchManager.cs

```diff
@@ -109,12 +119,45 @@ public bool ApplyChangedPatchSettingsIfNeeded(Patch currentPatch)
 
         // Change the lighting
         UpdateLight(currentPatch.BiomeTemplate);
-
-        compoundCloudSystem.SetBrightnessModifier(currentPatch.BiomeTemplate.CompoundCloudBrightness);
+        compoundCloudBrightness = currentPatch.BiomeTemplate.CompoundCloudBrightness;
```

Comment:
I dislike how hard I need to think here "does compoundCloudBrightness variable get properly sent to the system" now with this change. Could this be done as a private property instead or maybe this variable could be removed entirely and instead all the methods would call into compoundCloudSystem directly?